dist/tools/ci/build_and_test.sh

```diff
@@ -43,7 +43,7 @@ function run {
     OUT_LENGTH="$(echo -n $OUT | wc -c)"
```

Comment:
Unrelated to this PR, but this whole OUT_LENGTH and `if` part should just be changed to 
```sh
if [ -n "$OUT" ]; then
```